Gems/RecastNavigation/Code/CMakeLists.txt

```
@@ -49,12 +49,35 @@ set_property(
     PROPERTY COMPILE_OPTIONS ${compile_options}
 )
 unset(compile_options)
+
+# Place external recast libraries under RecastNavigation gem.
 set(RECAST_FETCHCONTENT_FOLDER "Gems/RecastNavigation/External")
-set_property(TARGET DebugUtils PROPERTY FOLDER ${RECAST_FETCHCONTENT_FOLDER})
-set_property(TARGET Detour PROPERTY FOLDER ${RECAST_FETCHCONTENT_FOLDER})
-set_property(TARGET DetourCrowd PROPERTY FOLDER ${RECAST_FETCHCONTENT_FOLDER})
-set_property(TARGET DetourTileCache PROPERTY FOLDER ${RECAST_FETCHCONTENT_FOLDER})
-set_property(TARGET Recast PROPERTY FOLDER ${RECAST_FETCHCONTENT_FOLDER})
+set(recastLibraries DebugUtils;Detour;DetourCrowd;DetourTileCache;Recast)
+foreach(recastLibrary ${recastLibraries})
+    set_property(TARGET ${recastLibrary} PROPERTY FOLDER ${RECAST_FETCHCONTENT_FOLDER})
+endforeach()
+
+# Export external recast navigation targets for installers.
+set_property(DIRECTORY APPEND PROPERTY O3DE_SUBDIRECTORY_INSTALL_CODE [[
+    set(recastLibraries DebugUtils;Detour;DetourCrowd;DetourTileCache;Recast)
+    foreach(recastLibrary ${recastLibraries})
+        add_library(RecastNavigation::${recastLibrary} STATIC IMPORTED GLOBAL)
+
+        set_target_properties(RecastNavigation::${recastLibrary} 
+            PROPERTIES 
+                IMPORTED_LOCATION 
+                    "${LY_ROOT_FOLDER}/lib/${recastLibrary}${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

You should add ${CMAKE_STATIC_LIBRARY_PREFIX} as well, since on non-Windows libraries start with the prefix of "lib"
```suggestion
                    "${LY_ROOT_FOLDER}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}${recastLibrary}${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

---

I'll hop to Linux and test it out there.